download/interface.go

```diff
@@ -0,0 +1,20 @@
+package download
+
+import (
+	"context"
+
+	"github.com/open-policy-agent/opa/bundle"
+)
+
+type DownloaderInterface interface {
+	Trigger(ctx context.Context) error
```

Comment:
[nit] let's drop the arg names where they don't add anything:
```suggestion
	Trigger(context.Context) error
```
In `SetCache(etag string)`, it makes sense to keep it 👍